Code/Framework/AzFramework/AzFramework/Spawnable/Script/SpawnableScriptBus.h

```diff
@@ -21,6 +21,7 @@ namespace AzFramework::Scripts
         // EBusTraits overrides
         using BusIdType = EntitySpawnTicket::Id;
         static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
```

Comment:
You can make both these constants `constexpr`
```suggestion
        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
        static constexpr AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
```

Reply:
I will include it in next the PR since I already have AR green, thanks